src/test/test_tls.c

```diff
@@ -0,0 +1,507 @@
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
I also had to add `#include <stdint.h>` (`uint8_t` definitions) for it to compile on Linux.

Reply:
Will add it